surfer/tests/test_viz.py

```diff
@@ -191,16 +191,26 @@ def test_meg_inverse():
     stc = io.read_stc(stc_fname)
     data = stc['data']
     vertices = stc['vertices']
-    time = 1e3 * np.linspace(stc['tmin'],
-                             stc['tmin'] + data.shape[1] * stc['tstep'],
-                             data.shape[1])
+    time = np.linspace(stc['tmin'], stc['tmin'] + data.shape[1] * stc['tstep'],
+                       data.shape[1])
```

Comment:
The endpoint needs to be `stc['tmin'] + (data.shape[1] - 1) * stc['tstep']` right?